src/extension.ts

```diff
@@ -48,6 +49,17 @@ export async function activate(context: vscode.ExtensionContext) {
 		}
 	}, null, context.subscriptions));
 
+	// Make the snippets available in arb files
+	const completions = getSnippets();
+	context.subscriptions.push(vscode.languages.registerCompletionItemProvider(
+		{ language: 'json', pattern: `**/*.arb` },
+		{
+			provideCompletionItems(document, position, token, context) {
```

Comment:
Not for this PR, but a possible future enhancement - since you can parse the file in the extension, you could figure out whether the cursor is inside a string, at the top level, etc. and filter the snippets to only include those that are relevant in that context.

Reply:
Yes! I would have to integrate it with the JSON parser though, I would not want to parse the file twice...

Reply:
Yep! I suspect other features (like fixes) will probably depend on parsing too. It could make sense to parse it once into some sort of mini AST structure that can easily be used by multiple other components (that could also allow the logic for errors/decorations to be decoupled a bit).